draft-ietf-quic-transport.md

```diff
@@ -4188,115 +4191,119 @@ ACK Delay:
   larger range of values with a shorter encoding at the cost of lower
   resolution.
 
-ACK Block Count:
+ACK Range Count:
 
-: A variable-length integer specifying the number of Additional ACK Block (and
-  Gap) fields after the First ACK Block.
+: A variable-length integer specifying the number of Gap and ACK Range fields in
+  the frame.
 
-ACK Blocks:
+First ACK Range:
 
-: Contains one or more blocks of packet numbers which have been successfully
-  received, see {{ack-block-section}}.
+: A variable-length integer indicating the number of contiguous packets
+  preceding the Largest Acknowledged that are being acknowledged.
```

Comment:
I think some more text about parsing this needs to be moved up. How about adding something like: 

"The smallest value in this range is determined by the formula:
~~~
   smallest = Largest Acknowledged - First ACK Range
~~~

The First ACK range acknowledges all packet numbers between the smallest and the Largest Acknowledged, inclusive."